src/tools.ts

```diff
@@ -340,8 +341,8 @@ export const registerTools = async (server: McpServer): Promise<void> => {
   server.tool(
     "updateEnvironment",
     `the updateEnvironment tool can update an environment for a given test target.
-    an environment represents a specific setup or deployments for a test target. It include a test account when necsesary
-    to login, a header configuration, a discovery url and a set of variables.`,
+    An environment represents a specific setup or deployments for a test target. It includes a test account when necessary
```

Comment:
typos fixed and removed mention of variables THAT ARE NOT updateable according to the schema